DTS-STN/Scrum-Poker
# DUMS 90 Set user color

## [DUMS-90](https://jira-dev.bdm-dev.dts-stn.com/browse/DUMS-90) (Assigning a color to the user)

### Description
Assigns a random color to each of the users (except current user) 

List of proposed changes:
- Sheila changed the user list spacing
- Color Array with figma colors
- Sets the State with the color for each user
- updated missing t prop on second player, currently missing on main 

### What to test for/How to test
- Open two browsers 
- create a room 
- join the room on the second browser
- see the colors for each user

### Additional Notes
